starboard/android/shared/media_codec_bridge_eradicator.cc

```diff
@@ -27,6 +27,12 @@ namespace shared {
 SB_ONCE_INITIALIZE_FUNCTION(MediaCodecBridgeEradicator,
                             MediaCodecBridgeEradicator::GetInstance);
 
+// Maximum wait time when creating a new MediaCodecBridge if the background
+// cleanup thread (MediaCodecBridgeEradicator::DestroyMediaCodecBridge) is
+// unresponsive.
+const int64_t ERADICATOR_THREAD_TIMEOUT_MICROSECOND =
```

Comment:
@xiaomings I choose 2 seconds here, it may help with the ANR issues on OMX devices. ANR cut at 5 seconds.

After a user seek on a video, if the old MediaCodecBridge's destructor is still running after 2 seconds, we will just try to create a new instance and let the video continue to play.